addons/misc/CfgVehicles.hpp

```diff
@@ -166,9 +166,266 @@ class CfgVehicles {
             };
         };
     };
-
+    class Items_base_F;
+    class Kat_armbandRC: Items_base_F {
+        scope = 2;
+        scopeCurator = 2;
+        scopeArsenal = 2;
+        author = "Miss Heda, vccv9040 (Swedish Forces Pack)";
+        editorPreview = QPATHTOF(ui\ArmbandWhiteCross.paa);
+        model = QPATHTOF(models\armband\Armband.p3d);
+        hiddenSelections[] = {"Armband"};
+        hiddenSelectionsTextures[] = {QPATHTOF(models\armband\ArmbandWC.paa)};
+    };
+    class Kat_armbandRCM: Kat_armbandRC {
+        author = "Miss Heda, vccv9040 (Swedish Forces Pack)";
+        editorPreview = QPATHTOF(ui\ArmbandWhiteCrossMedic.paa);
+        model = QPATHTOF(models\armband\Armband.p3d);
+        hiddenSelectionsTextures[] = {QPATHTOF(models\armband\ArmbandWCM.paa)};
+    };
+    class Kat_armbandRCD: Kat_armbandRC {
+        author = "Miss Heda, vccv9040 (Swedish Forces Pack)";
+        editorPreview = QPATHTOF(ui\ArmbandWhiteCrossDoctor.paa);
+        model = QPATHTOF(models\armband\Armband.p3d);
+        hiddenSelectionsTextures[] = {QPATHTOF(models\armband\ArmbandWCD.paa)};
+    };
+    class Kat_armbandKAT: Kat_armbandRC {
+        author = "Miss Heda, vccv9040 (Swedish Forces Pack)";
+        editorPreview = QPATHTOF(ui\ArmbandKat.paa);
+        model = QPATHTOF(models\armband\Armband.p3d);
+        hiddenSelectionsTextures[] = {QPATHTOF(models\armband\ArmbandKAT.paa)};
+    };
     class Man;
     class CAManBase: Man {
+        class ACE_SelfActions
+        {
+            class ACE_Equipment
+            {
+                class Armband {
+                    displayName = CSTRING(Armband_Sling);
```

Comment:
so just general - most of stringtables are broken because you use Sling_X, without Armband